<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myNotepad</class>
 <widget class="QMainWindow" name="myNotepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myNotepad</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>搜索(&amp;S)</string>
    </property>
    <addaction name="action_SearchNext"/>
    <addaction name="action_Replace"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_O">
     <property name="title">
      <string>编码(&amp;O)</string>
     </property>
     <addaction name="actionGB18030"/>
     <addaction name="actionUTF_8"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>字体大小
</string>
     </property>
     <addaction name="action16"/>
     <addaction name="action30"/>
     <addaction name="action60"/>
     <addaction name="action72"/>
    </widget>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Delete"/>
    <addaction name="action_SelectAll"/>
    <addaction name="separator"/>
    <addaction name="menu_O"/>
    <addaction name="menu_2"/>
    <addaction name="action_AutoChangeLine"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>改变背景</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
     <addaction name="action3"/>
    </widget>
    <addaction name="action_Close"/>
    <addaction name="action_CloseAll"/>
    <addaction name="separator"/>
    <addaction name="action_Cascade"/>
    <addaction name="action_Tile"/>
    <addaction name="separator"/>
    <addaction name="action_Previous"/>
    <addaction name="action_Next"/>
    <addaction name="separator"/>
    <addaction name="menu"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_S"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="action_SaveAs"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Previous"/>
   <addaction name="action_Next"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/new.png</normaloff>:/new/prefix1/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/open.png</normaloff>:/new/prefix1/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/save.png</normaloff>:/new/prefix1/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/saveAs.png</normaloff>:/new/prefix1/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/exit.png</normaloff>:/new/prefix1/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_SearchNext">
   <property name="text">
    <string>查找(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/undo.png</normaloff>:/new/prefix1/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/redo.png</normaloff>:/new/prefix1/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/cut.png</normaloff>:/new/prefix1/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪贴(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/copy.png</normaloff>:/new/prefix1/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/paste.png</normaloff>:/new/prefix1/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
  </action>
  <action name="action_SelectAll">
   <property name="text">
    <string>全选(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGB18030">
   <property name="text">
    <string>GB18030</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="text">
    <string>全部关闭(&amp;A)</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="text">
    <string>平铺(&amp;C)</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>层叠(&amp;T)</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/previous.png</normaloff>:/new/prefix1/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>上一个(&amp;P)</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/next.png</normaloff>:/new/prefix1/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;N)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/about.png</normaloff>:/new/prefix1/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action30">
   <property name="text">
    <string>30</string>
   </property>
  </action>
  <action name="action60">
   <property name="text">
    <string>60</string>
   </property>
  </action>
  <action name="action72">
   <property name="text">
    <string>72</string>
   </property>
  </action>
  <action name="action_AutoChangeLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
